tools/release.sh

```diff
@@ -86,7 +86,7 @@ SetVersion() {
     done
     sed -i "s/^__version.*/__version__ = '$ver'/" letsencrypt/__init__.py
 
-    git add -p $SUBPKGS # interactive user input
+    git add -p letsencrypt $SUBPKGS # interactive user input
```

Comment:
Actually letsencrypt is already present here: https://github.com/letsencrypt/letsencrypt/pull/1978/files#diff-b91eeb35e4e3920b1a96d2d03212320aR45